extensions/wrapper/clients/typescript-client/src/oauth2/AccessTokenStore.ts

```diff
@@ -0,0 +1,25 @@
+import {AccessTokenFetcher} from './AccessTokenFetcher';
+
+export class AccessTokenStore {
```

Comment:
the only thing i am confused still about is how this is a store, and not the fetcher, it does some request caching, but also token caching? gotta see about that tomorrow